src/discussions/discussions-home/DiscussionsHome.jsx

```diff
@@ -92,67 +97,89 @@ const DiscussionsHome = () => {
               })}
             >
               {!enableInContextSidebar && (
-                <NavigationBar />
+              <NavigationBar />
               )}
               <PostActionsBar />
             </div>
             <DiscussionsRestrictionBanner />
           </div>
+          )}
+
           {provider === DiscussionProvider.LEGACY && (
-            <Suspense fallback={(<Spinner />)}>
-              <Routes>
-                {[
-                  ROUTES.TOPICS.CATEGORY,
-                  ROUTES.TOPICS.CATEGORY_POST,
-                  ROUTES.TOPICS.CATEGORY_POST_EDIT,
-                  ROUTES.TOPICS.TOPIC,
-                  ROUTES.TOPICS.TOPIC_POST,
-                  ROUTES.TOPICS.TOPIC_POST_EDIT,
-                ].map((route) => (
-                  <Route
-                    key={route}
-                    path={route}
-                    element={<LegacyBreadcrumbMenu />}
-                  />
-                ))}
-              </Routes>
-            </Suspense>
+          <Suspense fallback={(<Spinner />)}>
+            <Routes>
+              {[
+                ROUTES.TOPICS.CATEGORY,
+                ROUTES.TOPICS.CATEGORY_POST,
+                ROUTES.TOPICS.CATEGORY_POST_EDIT,
+                ROUTES.TOPICS.TOPIC,
+                ROUTES.TOPICS.TOPIC_POST,
+                ROUTES.TOPICS.TOPIC_POST_EDIT,
+              ].map((route) => (
+                <Route
+                  key={route}
+                  path={route}
+                  element={<LegacyBreadcrumbMenu />}
+                />
+              ))}
+            </Routes>
+          </Suspense>
           )}
-          <div className="d-flex flex-row position-relative">
-            <Suspense fallback={(<Spinner />)}>
-              <DiscussionSidebar displaySidebar={displaySidebar} postActionBarRef={postActionBarRef} />
-            </Suspense>
-            {displayContentArea && (
+          {(courseStatus === LOADED || enableInContextSidebar) && (
+          <div>
+            { isEnrolled === false ? (
```

Comment:
I think instead of using ternary operator here conditional rendering could be a better approach here, also more readable 
{ isEnrolled && code....}
{!isEnrolled && code...}

Reply:
This is intentional as per my discussion with Awais, to avoid multiple checks.